test/compiler/EscapeAnalysis/interprocedural.jl

```diff
@@ -36,21 +36,21 @@ end
 let result = code_escapes((SafeRef{String},); optimize=false) do x
         return identity(x)
     end
-    r = only(findall(isreturn, result.ir.stmts.inst))
+    r = only(findall(isreturn, result.ir.stmts.stmt))
```

Comment:
@aviatesk once this is in, could we factor some of these predicates out into the irutils?

Reply:
Yeah. We'd even want to define them within `Core.Compiler` so that external packages can use them too?